actors/evm/tests/asm.rs

```diff
@@ -0,0 +1,55 @@
+use etk_asm::ingest::Ingest;
+use evm::interpreter::opcode::OpCode::*;
+use fil_actor_evm as evm;
+
+#[allow(dead_code)]
+pub fn new_contract(name: &str, init: &str, body: &str) -> Result<Vec<u8>, etk_asm::ingest::Error> {
```

Comment:
Just some docs that indicate that this produces the initcode for the supplied EVM bytecode as-is would be great.

Reply:
nit: should we call this `new_initcode` so that it's extremely clear what it does?

Reply:
yeah, i'll add some docs.

We can rename it, but really this is what we understand as the contract in our land.

I wont object to renaming it if you feel strongly about it.

Reply:
added docs, left the name as is for now.